### DISCUSSION TITLE
Can't rewrite "/" to "new/path"

### DISCUSSION THREAD

**Discussion by meotimdihia** [vercel/next.js#17513] [Help] [answered]
It seems we can't rewrite homepage "/":

**Work:**
```
{
    async rewrites() {
      return [
        {
          source: '/about',
          destination: '/schedule/2020/fall',
        },
      ]
    }
}
```

**Not work:**
```
{
    async rewrites() {
      return [
        {
          source: '/',
          destination: '/schedule/2020/fall',
        },
      ]
    }
}
```

**Comment by ijjk** [comment, marked as answer]
Hi, do you have an index file in your pages directory? Rewrites aren't able to overwrite a page or public file as they take higher priority. If you do not have a `pages/index.${EXT_NAME}` file and the rewrite is still not working then we can investigate this further if you provide a repo with a minimal reproduction.

**Reply by meotimdihia** [reply]
Thanks. I removed the index file in my pages directory and it works now. 😀
